assets/styles/layout/_options.scss

```diff
@@ -113,6 +140,10 @@
     scrollbar-width: none;
   }
 
+  &__filter {
+    justify-content: start;
+  }
```

Comment:
please put this in the other `__filter` line 88

Reply:
Also done 👍🏻👍🏻

Reply:
Actually, wait - moving this breaks the options bar. My guess is that "justify-content: end" from the "& > *" block will override it if this block doesn't come later in the file.

I was mainly just following the format of where __main and __view were defined. I consolidated all the button settings as requested, and moved this one to a little less random place while still leaving it late in the file so it'll take effect.